app-frontend/src/app/pages/library/scenes/detail/detail.html

```diff
@@ -1,16 +1,21 @@
 <div class="row content stack-sm">
-  <div class="column-8" ng-show="!$ctrl.loading">
+  <div class="column-8" ng-if="!$ctrl.loading">
     <div class="embedded-scrollable">
       <h1 class="h3 page-title">
         <a ui-sref="library.scenes.list">{{$ctrl.scene.id}}</a>
```

Comment:
I don't think we want the UUID to be exposed to the user in the UI. It's not significant to them and doesn't make sense. The scene name should be there below scenes I think.